README.md

```diff
@@ -1,87 +1,77 @@
+All of this operates under a  different concept than the one we all have become used to in recent times. 
 
-We like to think this is not done by re-inventing the wheel, but by making sure the road is clean, leveled and with less obstacles
-than before, by removing some heavy weight from the top, some unneeded ballast, plus, of course, some well positioned innovations to make the ride enjoyable. 
+We like to think this is not done by re-inventing the wheel, but by making sure the road is clean, level, and with fewer obstacles
+than before. We do this by removing some heavy weight from the top, some unneeded ballast, plus, of course, some well positioned innovations to make the ride enjoyable. 
 
-We also like to say that Archipelago is like a Metadata Synthetizer (LFO anyone?) and we want to give you all the knobs, parameters, inputs and outputs to make the best out of it. 
-Still you can make "music" by just tapping the keyboard. 
+We also like to say that Archipelago is like a Metadata Synthetizer (LFO anyone?) and we want to give you all the knobs, parameters, inputs and outputs to make the best out of it. Still, you can make "music" by just tapping the keyboard. 
 
-Finally. It tries to stay humble, slim and nimble in nature with a small code base full of inline comments and `@todos`. 
+Finally, Archipelago tries to stay humble, slim, and nimble in nature with a small code base full of inline comments and `@todos`. 
 
-To get here we had to do a full stop first. Look around. Questioning everything we knew. Research and test (repeat) and then re-Architect slowly 
-on new and old assumptions, but specially new community values. 
+To get here we had to do a full stop first. Look around. Questioning everything we knew. Research and test (repeat) and then re-architect slowly on new and old assumptions, and especially new community values. 
 
 Learn [`what makes Archipelago so different`](docs/ourtake.md) and hopefully also a well known place.
 
 
-## Strawberryfields for ever
+## Strawberryfields Forever
 
 Archipelago integrates transparently into the Drupal 8 ecosystem using its Core Content Entity System (Nodes), Discovery (Search API) and in general all its Core Components plus a few well maintained external ones. 
 
-But by design (and because we think its imperative) it takes fully charge, owns, the Metadata layer and their associated Media assets by implementing a highly
-configurable and also smart JSON based Drupal field named [`strawberryfield`](https://github.com/esmero/strawberryfield/tree/8.x-1.0-beta1) that attaches to any Content. 
+By design (and because we think its imperative), Archipelago takes full charge of the metadata layer and associated media assets by implementing a highly configurable, smart JSON-based Drupal field named [`strawberryfield`](https://github.com/esmero/strawberryfield/tree/8.x-1.0-beta1) that attaches to any content. 
```

Comment:
Here i would like to keep the work JSON separated from the based, is that grammatically incorrect? Not a strong opinion but some buzzwords, core to Archipelago's architecture i feel live better by them selves.

Reply:
I read "JASON-based" as a compound noun; including a hyphen is grammatically correct. Perhaps we could consider alternate phrasing for this. Or we can just ignore the rules. 😉